crates/bevy_ecs/src/system/commands.rs

```diff
@@ -223,6 +223,13 @@ pub struct Commands {
 }
 
 impl Commands {
+    /// Creates a new entity with a given component in the current [World].
```

Comment:
This is incorrect. This doesn't create an entity with a given component, but rather with a given DynamicBundle (which is a "set" of components).

It is worth showing two examples here: a spawn example using tuples:

```rust
commands.spawn((MyComponentA, MyComponentB))
```

And an example using a derived Bundle

```rust
#[derive(Bundle)]
struct MyBundle {
  a: MyComponentA,
  b: MyComponentB,
}

commands.spawn(MyBundle {
  a: MyComponentA,
  b: MyComponentB,
})
```

Reply:
And lets call out that the two examples are equivalent.